### PR TITLE
Update the documentation style

### PR DESCRIPTION
**Pull request by chitrakotwani**
- Add expired profile section
- Made first half of document pithy

### CODE REVIEW

File: Cookbook/Technical-Documents/FastlaneMatch.md
Comment (karabatov) [inline Cookbook/Technical-Documents/FastlaneMatch.md]:
```suggestion
Note:
```

**Review by karabatov — APPROVED** [review]

File: Cookbook/Technical-Documents/FastlaneMatch.md
Comment (zzcgumn) [inline Cookbook/Technical-Documents/FastlaneMatch.md]:
```
The reason to specify individual app identifiers is that otherwise if AppStore bundle ids are added to the default set of identifiers Match will try to load certificates that may not exist for some apps on the portal, as those apps are registered in different teams.
```
->
```
We specify individual app identifiers instead of adding all App Store bundle ids to the default set as Match would otherwise attempt to load certificates that do not exist in the portal. This is because some of our app ids are registered with teams that belong to our partners.
```

File: Cookbook/Technical-Documents/FastlaneMatch.md
Comment (zzcgumn) [inline Cookbook/Technical-Documents/FastlaneMatch.md]: Please add the date here. Otherwise I will not know whether it is worth checking if the fastlane has learned how to renew expired profiles.

File: Cookbook/Technical-Documents/FastlaneMatch.md
Comment (zzcgumn) [inline Cookbook/Technical-Documents/FastlaneMatch.md]:
```suggestion
- Register device manually on the portal in the enterprise team account
```

**Review by zzcgumn — CHANGES_REQUESTED** [review]
Thank you so much for documenting how upgrading profiles work together with Fastlane Match. 

Before merging we should fix some typos and add a date to make the at the time of writing well defined.

I am also suggesting that we rephrase a paragraph in a way that I think makes it easier to read, but feel free to disregard that comment if you disagree with me.

File: Cookbook/Technical-Documents/FastlaneMatch.md
Comment (chitrakotwani) [reply Cookbook/Technical-Documents/FastlaneMatch.md]: Nice. I was struggling to phrase it nicely myself :)

Comment (chitrakotwani) [inline Cookbook/Technical-Documents/FastlaneMatch.md]:
```suggestion
We specify individual app identifiers instead of adding all App Store bundle ids to the default set as Match would otherwise attempt to load certificates that do not exist in the portal. This is because some of our app ids are registered with teams that belong to our partners.
```

**Comment by chitrakotwani** [comment]
@zzcgumn  - Thanks a lot for your comments. They have been addressed now . :)

**Review by zzcgumn — APPROVED** [review]

File: Cookbook/Technical-Documents/FastlaneMatch.md
Comment (AliSoftware) [inline Cookbook/Technical-Documents/FastlaneMatch.md]: I'd rather use the same placeholder names and syntax throughout the whole document. So replacing `<appstore team id>` with `((Team-ID}}` (nit: I even prefer `{{TeamID}}` without the dash), and use always the same delimiters (either curly braces or angle brackets but not a mix of both)

**Comment by AliSoftware** [comment]
Whoops I see I forgot to hit "Submit" on my pending review from this morning and it's too late already 😅 I still think it could be nice to uniformize and mention the AppStore team in the table 😉

File: Cookbook/Technical-Documents/FastlaneMatch.md
Comment (chitrakotwani) [reply Cookbook/Technical-Documents/FastlaneMatch.md]: I did not want to mention app-store team there because these are handled specially as described in next section. I did not want a developer to read first section and run the command with just a couple of options

File: Cookbook/Technical-Documents/FastlaneMatch.md
Comment (chitrakotwani) [reply Cookbook/Technical-Documents/FastlaneMatch.md]: Good observation. I had left the previous commands pretty much untouched and for the ones I wrote, I used curly braces

File: Cookbook/Technical-Documents/FastlaneMatch.md
Comment (chitrakotwani) [reply Cookbook/Technical-Documents/FastlaneMatch.md]: :D Let me fix that

File: Cookbook/Technical-Documents/FastlaneMatch.md
Comment (AliSoftware) [reply Cookbook/Technical-Documents/FastlaneMatch.md]: Ok, make sense, but then I think the wording is confusing.

Maybe find a place upstream, like a paragraph used as introduction of them all, to say:

> We use two teams on our portal:
> - one "Enterprise" team we use for Day to Day developent and HockeyApp (see <link to paragraph>)
> - one "AppStore" team used for TestFlight and AppStore (see <link to paragraph>)

Then in the specific paragraphs, just reference them, e.g.:
 - remove the "We have two developer teams – Enterprise and App-store" line from here (since you'd have moved it upstream in the intro) but instead say "For day to day and HockeyApp, we use certificates and profiles from our Development team"
 - in the paragraph about TestFlight and AppStore, put a similar paragraph like "For TestFlight and AppStore, we use certificates and profiles from our App Store developer team"

**Pull request merged**
